src/data/accessibility.json

```diff
@@ -0,0 +1,55 @@
+{
+    "meta": {
+        "title": "Accessibility statement by the Robert Koch Institute and the Federal Ministry of Health",
```

Comment:
Suggest to change this to title case i.e.

"statement" => "Statement"

Reply:
Fixed in https://github.com/corona-warn-app/cwa-website/pull/2508/commits/4799ce6137f4737239b709eada61369ca3d85ee8, thanks!